docs/apis-tools/migration-manuals/migrate-to-camunda-user-tasks.md

```diff
@@ -329,7 +324,7 @@ The following table outlines the respective endpoints. Click the endpoints to fo
     </tr>
     <tr>
         <th style={{ textAlign: "end" }}>Update task</th>
-        <td style={{color: "gray"}}>-</td>
+        <td style={{color: "gray"}}>Not supported</td>
```

Comment:
Non-blocking: One for the tech-writing team - do we say "unsupported" or "not supported"?

Reply:
I think not supported is fine in these instances.

Reply:
Unsupported is marginally smaller, so can be better in a table of supported/unsupported items, but it is quite a niche case - we can discuss in a tech sync 👍